client/components/AddSponsor.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+
+interface AddSponsorProps {
+  eventId: string;
+  chapterId: string;
+}
+
+interface SponsorData {
+  name?: string;
+  website?: string;
+  type?: string;
```

Comment:
since the possible values for this is known we should just make this an enum:

```
type: 'FOOD' | 'BEVERAGE' | 'OTHER';
```

Reply:
Agree and fixed. I originally did try and do by declaring an enum SponsorTypes and then trying to reference that but I couldn't get it to work.